Documentation/ceph-upgrade.md

```diff
@@ -265,7 +268,11 @@ Let's get started!
 > instructions to [migrate the Drive Group spec](#migrate-the-drive-group-spec) before performing
 > any of the upgrade steps below.
 
-## 1. Update common resources and CRDs
+## 1. Refer to the updates section for ancillary resources
```

Comment:
How about if this were a subsection of the next section? Since it's not enabled by default, it doesn't seem critical enough to list it first.